test/parallel/test-http-outgoing-renderHeaders.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const common = require('../common');
+const assert = require('assert');
+
+const outHeadersKey = require('internal/http').outHeadersKey;
```

Comment:
This will probably need to have `// Flags: --expose-internal` added on the line below `'use strict';`